graphql-server/metadata/tables.yaml

```diff
@@ -49,8 +49,12 @@
             players:
               id:
                 _eq: X-Hasura-User-Id
-        - player_id:
-            _eq: X-Hasura-User-Id
+        - _or:
+          - player_id:
+              _eq: X-Hasura-User-Id
+          - game:
+              host_id:
+                _eq: X-Hasura-User-Id
```

Comment:
you can update the card if it's yours or you're the host (because the host will need to set is_allowed)